src/common/components/dropdown/RHFDropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import { ForwardRefRenderFunction, forwardRef } from "react";
+import { useController, Control } from "react-hook-form";
+import { useSelect } from "downshift";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAngleUp, faAngleDown } from "@fortawesome/free-solid-svg-icons";
+import {
+  DropdownContainer,
+  DropdownHeader,
+  DropdownList,
+  DropdownListItem,
+  UpArrow,
+  DownArrow,
+} from "./Dropdown.styled";
+
+interface OptionType {
+  value: string;
+  label: string;
+}
+
+interface Props {
+  name: string;
+  items: OptionType[];
+  control: Control;
```

Comment:
just curious, this `control` prop isnt used, what is it meant for?

Reply:
I'm not 100% sure, but react-hook-form throws a type error if it's not defined in the props.